renku/api/datasets.py

```diff
@@ -157,7 +157,7 @@ def add_data_to_dataset(
 
         dataset.files.update(files)
 
-    def _add_from_url(self, dataset, path, url, nocopy=False, **kwargs):
+    def _add_from_url(self, dataset, path, url, is_link=False, **kwargs):
```

Comment:
what is the purpose of `is_link`? maybe it should be called `link`.

Reply:
is_ or at_ and similar prefixes/suffixes I'm using to denote certain types (like booleans or datetime). It's a matter of styled I used before, some habits die hard. I can rename if it bothers you.